cli/options.js

```diff
@@ -88,6 +92,11 @@ function getCliOptions (projectPath) {
     .wrap(Math.min(150, yargs.terminalWidth()))
     .argv
 
+  if (options.bindAddress) {
+    console.warn('The use of --bindAddress is deprecated. Use the --address option instead.')
```

Comment:
Should I use `console.warn` to log the deprecation warning, or is there another logging utility that I should use?

If I should use `console.warn`, any idea how to mock it with `simple-mock` so the output doesn't display when testing?